### PR TITLE
Add holoAng(n), holoPos(n) getters

### PR DESCRIPTION
**Pull request by bjcscat**
Add functions to simplify getting holo angles and positions

### CODE REVIEW

**Review by thegrb93 — APPROVED** [review]

**Review by Denneisk — APPROVED** [review]

**Comment by Vurv78** [comment]
Merged too fast. This was missing `nodiscard` and I would've preferred the `Angle(ang[1],ang[2],ang[3])` be converted to just `ang`.

File: lua/entities/gmod_wire_expression2/core/hologram.lua [hunk 1]
```diff
@@ -1003,13 +1003,27 @@ e2function void holoPos(index, vector position)
 	WireLib.setPos(Holo.ent, Vector(position[1],position[2],position[3]))
 end
 
+e2function vector holoPos(index)
+	local Holo = CheckIndex(self, index)
+	if not Holo then return end
```
Comment (Vurv78) [inline lua/entities/gmod_wire_expression2/core/hologram.lua:1008]: Missing return here

File: lua/entities/gmod_wire_expression2/core/hologram.lua [hunk 2]
```diff
@@ -1003,13 +1003,27 @@ e2function void holoPos(index, vector position)
...
 e2function void holoAng(index, angle ang)
 	local Holo = CheckIndex(self, index)
 	if not Holo then return end
 
 	WireLib.setAng(Holo.ent, Angle(ang[1],ang[2],ang[3]))
 end
 
+e2function angle holoAng(index)
+	local Holo = CheckIndex(self, index)
+	if not Holo then return end
```
Comment (Vurv78) [inline lua/entities/gmod_wire_expression2/core/hologram.lua:1022]: Missing return here :\

Comment (thegrb93) [reply lua/entities/gmod_wire_expression2/core/hologram.lua:1022]: It was consistent with the other holoAng so it looked fine to me. Both should be updated.

Comment (Derpius) [reply lua/entities/gmod_wire_expression2/core/hologram.lua:1022]: That other method is a void function, this returns an angle

File: lua/entities/gmod_wire_expression2/core/hologram.lua (hunk 1 above)
Comment (Denneisk) [reply lua/entities/gmod_wire_expression2/core/hologram.lua:1008]: Your analysis is greatly appreciated. Maybe for future reference you should make a note in the style guide or somewhere else that attribute should be used whenever possible.

File: lua/entities/gmod_wire_expression2/core/hologram.lua (hunk 2 above)
Comment (thegrb93) [reply lua/entities/gmod_wire_expression2/core/hologram.lua:1022]: Ah I see

**Pull request merged**